packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -121,8 +122,10 @@ module.exports = class AwsS3Multipart extends Plugin {
 
     const filename = encodeURIComponent(key)
     const uploadIdEnc = encodeURIComponent(uploadId)
-    return this.client.post(`s3/multipart/${uploadIdEnc}/complete?key=${filename}`, { parts })
-      .then(assertServerError)
+    return this.client.post(`s3/multipart/${uploadIdEnc}/complete?key=${filename}`, {
+      parts,
+      metadata: file.meta
```

Comment:
S3 multipart uploads need metadata to be assigned when the upload is _created_, not when it's complete. it should not be necessary to add it here